packages/swap/src/routes/swap/index.ts

```diff
@@ -35,7 +35,8 @@ type SwapWithBroadcast = Swap & {
     | null;
 };
 
-const uuidRegex = /^[a-f\d]{8}(-[a-f\d]{4}){4}[a-f\d]{8}$/i;
+const channelIdRegex = /^(\d+)-([a-z]+)-([a-z]+)$/i;
```

Comment:
or we can use named capture groups to make it a bit clearer
```suggestion
const channelIdRegex = /^(?<blockId>\d+)-(?<chain>[a-z]+)-(?<channelId>\d+)$/i;
```

```ts
/^(?<blockId>\d+)-(?<chain>[a-z]+)-(?<channelId>\d+)$/i.exec('1234-ETH-2').groups
// => {blockId: '1234', chain: 'ETH', channelId: '2'}
```

Reply:
thats nice!